e2e/portalicious/pages/ProjectTeam.ts

```diff
@@ -5,11 +5,27 @@ import BasePage from './BasePage';
 class ProjectTeam extends BasePage {
   readonly page: Page;
   readonly tableRows: Locator;
+  readonly usersDropdown: Locator;
+  readonly chooseUserDropdown: Locator;
+  readonly chooseRoleDropdown: Locator;
+  readonly submitButton: Locator;
+  readonly removeUserButton: Locator;
 
   constructor(page: Page) {
     super(page);
     this.page = page;
     this.tableRows = this.page.locator('table tbody tr');
+    this.usersDropdown = this.page.getByRole('option');
+    this.chooseUserDropdown = this.page.locator(
+      `[formControlName="userValue"]`,
+    );
+    this.chooseRoleDropdown = this.page.locator(
+      `[formControlName="rolesValue"]`,
+    );
```

Comment:
I tried it but the label refers to something else and cannot distinguish the exact element I am looking for I think in this case we are using formControlName instead of label.